src/components/SearchMap/SearchMapWithGoogleMaps.js

```diff
@@ -466,22 +510,54 @@ class SearchMapWithGoogleMap extends Component {
   }
 
   render() {
-    const { onMapLoad, onMapMoveEnd, center, bounds, zoom, ...rest } = this.props;
-    const zoomAndCenter =
-      !bounds && !center ? { zoom: 1, center: { lat: 0, lng: 0 } } : { zoom, center };
+    const {
+      className,
+      onMapLoad,
+      onMapMoveEnd,
+      center,
+      bounds,
+      zoom,
+      listings,
+      activeListingId,
+      infoCardOpen,
+      onListingClicked,
+      mapComponentRefreshToken,
+      onListingInfoCardClicked,
+      createURLToListing,
+      ...rest
+    } = this.props;
     return (
-      <MapWithGoogleMap
-        onMapLoad={this.onMapLoad}
-        onIdle={this.onIdle}
-        bounds={bounds}
-        {...zoomAndCenter}
-        {...rest}
-      />
+      <div
+        id="map"
```

Comment:
I think this is a bit too prone to conflict. If the id is really required, could it come as a prop from the page?

Reply:
I'd keep the id because Google Maps is quite difficult to debug without some `document.getElementById()` calls from DevTools. (The element on those map panes are not selectable by right-clicking on top of the map.)

Anyway, I passed the id ("searchMap") from SearchMap component, but I didn't push it further than that. (And anyway, it's unlikely that there's more than one SearchMap instance since it takes 2/5 of the entire screen.

Reply:
Yes, it is unlikely that a page in the app will have multiple search maps. However, we do have plans to use the styleguide more for lots of things like map variants etc., so it would be helpful to be able to define unique IDs there.

Reply:
Showing these components, that take the entire-screen into use, needs some thinking.  However, the change I made now should allow different ids to be passed for the SearchMap component.